snuba/query/allocation_policies/__init__.py

```diff
@@ -199,6 +206,12 @@ def __init__(
                 value_type=int,
                 default=0,
             ),
+            AllocationPolicyConfig(
```

Comment:
This is not what I meant. I meant the maximum amount of threads the policy could allocate to a query.

Reply:
I created a `max_threads` property on the base class for this now, let me know if it makes sense.

As far as `throttled_thread_number` goes, I can revert it back to being on the bytes scanned class only but would it make sense for all policies to have it as well? I assume each policy would have to have it's own number it would throttle threads to regardless of what kind of policy it is?

Reply:
> I can revert it back to being on the bytes scanned class only but would it make sense for all policies to have it as well?

It does not make sense at this time. Policies will always throttle threads however it may not always be a simple constant value like the existing policy.